tests/common/dualtor/dual_tor_mock.py

```diff
@@ -339,7 +350,18 @@ def apply_mock_dual_tor_kernel_configs(request, tbinfo):
     '''
     Wraps all kernel related (routes and neighbor entries) fixtures for convenience
     '''
-    if tbinfo["topo"]["name"] == "t0":
+    if "t0" in tbinfo["topo"]["name"]:
         request.getfixturevalue("apply_dual_tor_peer_switch_route")
         request.getfixturevalue("apply_dual_tor_neigh_entries")
         logger.info("Done applying kernel configs for dual ToR mock")
+
+
+@pytest.fixture(scope="module")
+def cleanup_mocked_configs(duthost, tbinfo):
```

Comment:
Are you seeing any leftover configs that are not cleaned up properly after mocking? This shouldn't be necessary, and running `load_minigraph` would take a while.

Reply:
Yes, in the nightly test, `test_ipinip` caused `test_fdb` to fail due to the leftover from those mocking fixtures. @bingwang-ms, could you please elaborate on this?

Reply:
Sure. It was because the ```DEL``` operation of ```MUX_CABLE_TABL``` is not implemented in ```orchagent```. So when we are applying the ```DEL``` config, following logs will be observed
```
Mar 15 04:31:13.371252 str2-7050cx3-acs-01 NOTICE swss#swssconfig: :- main: Loading config from JSON file:/mux_standby_DEL.json...
Mar 15 04:31:13.373673 str2-7050cx3-acs-01 NOTICE swss#orchagent: :- delOperation: Deleting Mux state entry for port Ethernet4 not implemented
Mar 15 04:31:13.374644 str2-7050cx3-acs-01 NOTICE swss#orchagent: :- delOperation: Deleting Mux state entry for port Ethernet8 not implemented
Mar 15 04:31:13.375441 str2-7050cx3-acs-01 NOTICE swss#orchagent: :- delOperation: Deleting Mux state entry for port Ethernet12 not implemented
```
Although the config is deleted from config db, ```orchagent``` doesn't do the clean job. As suggested by @prsunny, a ```config reload``` is necessary to do a cleanup.

Reply:
Oh I see. In this case would `config reload` be faster or would `load minigraph`?